surveys/apps.py

```diff
@@ -0,0 +1,55 @@
+from collections import OrderedDict
+
+from django.apps import AppConfig
+from django.utils.translation import ugettext_lazy as _
+
+
+class SurveyConfig(AppConfig):
+    name = 'surveys'
+
+    # Complete dictionary of customisable data per field type.
+    # Has to be Ordered so we can guarantee consistent behaviour on things making
+    # use of it (dropdown fields etc).
+    field_type_data = OrderedDict((
+        ('free_text', {
+            'display_text': _('Character field (free text)'),
+            'form_field_factory': None,
```

Comment:
That seems like it wouldn't add much.  It also makes using `default_app_config` awkward.